store/pg_engine_test.go

```diff
@@ -16,47 +17,101 @@ import (
 	"go.uber.org/zap"
 )
 
-func TestGetMigrationVersions(t *testing.T) {
-	versions := []semver.Version{semver.MustParse("1.0.0"), semver.MustParse("1.1.0"), semver.MustParse("1.1.1"), semver.MustParse("1.2.0"), semver.MustParse("1.3.0")}
+func TestGetMinorMigrationVersions(t *testing.T) {
+	names := []string{latestDataFile, latestSchemaFile, "1.0.0", "1.1.0", "1.2.0", "1.3.0", "1.4.0"}
```

Comment:
My feeling is specifying 1.0.0/0001_foo.sql is more confusing than 1.0/0001_foo.sql, since 0001 is under 1.0 instead of under 1.0.0